config/config_rhel5.go

```diff
@@ -0,0 +1,31 @@
+//+build rhel5
+
+// Copyright (c) 2020 Sorint.lab S.p.A.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package config
+
+import (
+	"os"
+	"path/filepath"
+)
+
+// GetBaseDir return executable base directory, os independant
+func GetBaseDir() string {
+	s, _ := os.Readlink("/proc/self/exe")
+	s = filepath.Dir(s)
+
+	return s
+}
```

Comment:
Maybe here is a good idea to switch by windows/linux instead of switching by others/rhel5